# ENVIRONMENT SETTINGS
spawn_y: 0                    # Position world y
target_velocity: 15.0         # Target velocity for the robot (in meters per second)
direction: 1                  # 1=forward, -1=backwards on the road direction
stdev_landmark: 10.3334       # The standard deviation of the position of each landmark (in meters)
stdev_depth: 0.1              # The standard deviation of the depth from the depth sensor (in meters)
stdev_angle: 0.5235           # The maximum angle between the camera horizon and the landmark (in rad)
detection_rate: 1.0           # Probability of detecting a landmark

# SENSOR SETTINGS
accelerometer_noise_density: 0.083
accelerometer_random_walk: 0.00005
imu_update_rate: 100              